docs/tuf-spec.txt

```diff
@@ -928,24 +928,120 @@ Version 1.0 (Draft)
 
 5.1. The client application
 
-   1. The client application first instructs TUF to check for updates.
+    1. **Download the root metadata file**, up to some X number of bytes
+    (because the size is unknown). The value for X is set by the authors of the
+    application using TUF. For example, X may be tens of kilobytes.
 
-   2. TUF downloads and verifies timestamp.json.
+    1.1. **Check signatures.** The root metadata file must have been signed by
```

Comment:
Also, where in the specification do you think we should use "SHOULD" or "MUST" (ala RFC 2119)?